models/sushiswap/optimism/sushiswap_optimism_pool_incentives_rates.sql

```diff
@@ -0,0 +1,110 @@
+{{ config(
+    alias = 'pool_incentives_rates'
+    ,partition_by = ['block_date']
+    ,materialized = 'table'
```

Comment:
Hi @MSilb7, maybe this spell can be materialized as `view`?

Reply:
it probably can. My assumption was that `table` was better for runtime. Is that not correct?

Reply:
In my point of view, table may be suitable for static spells, and the server/warehouse stores all results (which accelerates the runtime performance). It will not update automatically when underlying data updates.

dbt docs link:
https://docs.getdbt.com/terms/table

Reply:
tables will be updated 1x/day, based on our orchestration schedule as of now